verify-push-backend/.env

```diff
@@ -2,4 +2,8 @@
 # format: sid
 # link: https://www.twilio.com/console/verify/services
 # required: true
-VERIFY_SERVICE_SID=
\ No newline at end of file
+VERIFY_SERVICE_SID=
+# description: Define if the identity field will be hashed or will be taken as it is, possible values hash or raw
+# format: text
+# required: false
+IDENTITY_PROCESSING=hash
```

Comment:
instead of using string values here could we make this a boolean and call it `HASH_IDENTITY` with a default of true?

Reply:
Or do we expect to have more than 2 values in the future?

Reply:
I started using boolean, but I am not sure which format should use: https://github.com/twilio-labs/configure-env/blob/main/docs/SCHEMA.md#format

Reply:
good catch, I asked about this on Slack! hopefully we can support this.